.github/workflows/ci_cd_pr.yml

```
@@ -117,7 +119,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: doc-style
     steps:
-      - uses: ansys/actions/doc-build@main
+      - uses: ansys/actions/doc-build@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/doc-build@main
```
Needs to be reverted before merging